### PR TITLE
[i18b sprint] 3760 translations loading

### PR DESCRIPTION
**Pull request by litvinovg**
**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues/3760)**
[VIVO PR](https://github.com/vivo-project/VIVO/pull/3788)
# What does this pull request do?
Replaces translation properties storage with semantic storage.
Configuration model to store semantic translations (associated directory name interface-i18n)
All code that became obsolete related to properties was removed.
# What's new?
TranslationProvider provides translations from triple store.
TranslationConverter converts found properties files into semantic data.
Note: translations in properties overwrites translations found in semantic storage.
VitroResourceBundle addAppPrefix was left for backwards compatibility.
# How to test
* Run unit tests
* Verify that switching between languages is working
* Try online translations (in developer panel settings)
* Try defeat cache of language property files  (in developer panel settings)
* Verify log the retrieval of language strings  (in developer panel settings)
* Try loading semantic data translations by putting them into directories:
interface-i18n/firsttime/
i18n/language_tag/interface-i18n/firsttime/
# Interested parties
@chenejac @brianjlowe

### CODE REVIEW

**Review by chenejac — COMMENTED** [review]
@litvinovg Please, check one my comment.

File: api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/TranslationProvider.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+	+ "   GRAPH <http://vitro.mannlib.cornell.edu/default/vitro-kb-applicationMetadata> {\n"
+	+ "	    ?portal vitro:themeDir ?themePath .\n"
+    + "     BIND(SUBSTR(?themePath, 8, STRLEN(?themePath) -8 ) as ?current_theme) .\n"
+	+ "	  }\n"
+	+ "	  BIND(COALESCE(?found_theme, \"none\") as ?theme ) .\n"
+	+ "	  FILTER(?theme = \"none\" || ?theme = ?current_theme) . "
+	+ "	  BIND(COALESCE(?found_application, \"none\") as ?application ) .\n"
+	+ "	  BIND(IF(?current_application = ?application && ?current_theme = ?theme, 3, "
+	+ "		   	IF(?current_theme = ?theme, 2, "
+	+ "				IF(?current_application = ?application, 1, 0)) ) AS ?order ) .\n"
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/TranslationProvider.java:57]: 3 case will be filtered, line 56 
 FILTER(?theme = \"none\" || ?theme = ?current_theme) . "

[the remark this answers is not in the document]
Comment (chenejac) [reply api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/TranslationProvider.java:57]: I see now, thanks

**Review by chenejac — APPROVED** [review]
Works perfect for me.

**Review by brianjlowe — APPROVED** [review]

**Pull request merged**